### ISSUE TITLE
build process for assets

### ISSUE THREAD

**Issue by UltraMC** [Tibia-WindBot/WindBot-Docs#55]
Currently builder creates html files from teplate and nothing more. If user wants to run docs locally he has no styles, scripts, graphics etc.

Shall that be added to build process?

**Comment by rmobis** [comment]
I'm pretty sure the `tibia-11` branch has the needed files for a local build. Could you check if they work for you?

**Comment by UltraMC** [comment]
They should share assets - are we ccreating separate repository/focusing only on 11/yoloing and copying/creating double changes?

**Comment by rmobis** [comment]
Nope, we just created a separate tibia-11 branch to add all tibia-11 functionality. We should have probably merged it back by now.

**Comment by UltraMC** [comment]
@rmobis please confirm before I'll start:
I can copy/cherrypick build process from tibia-11 to master

**Comment by rmobis** [comment]
I've merged tibia-11 branch into master.

**Comment by UltraMC** [comment]
PR https://github.com/Tibia-WindBot/WindBot-Docs/pull/58
